webui/react/src/components/Avatar.tsx

```diff
@@ -33,8 +33,21 @@ const Avatar: React.FC<Props> = ({ hideTooltip, name, large }: Props) => {
   const style = { backgroundColor: getColor(name) };
   const classes = [ css.base ];
   if (large) classes.push(css.large);
+
+  const [ photoSrc, setPhotoSrc ] = useState<string>('');
+  useMemo((): void => {
+    const img = new Image();
+    img.onload = () => {
+      setPhotoSrc(img.src);
+    };
+    img.src = `/users/${name}/image`;
+  }, [ name ]);
```

Comment:
curious - is this just a way to preload image so that `backgroundImage` doesn't load progressively?

Reply:
Many users won't have an image, so I was worried that setting backgroundImage would overwrite the backgroundColor and show a missing image icon. I don't see any issue in Chrome, Safari, or Firefox so I will make this much simpler